docs/UserGuide.md

```diff
@@ -230,17 +230,17 @@ add -n NAME -p PHONE_NUMBER -e EMAIL -rs RELATIONSHIP
 **Rules to Remember:**
 
 
-| Field       | Requirements                                                                                                                                                       |
-|-------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Name        | • Letters, numbers, and spaces only<br>• Cannot be blank                                                                                                           |
-| Phone       | • Minimally 3 digits<br>•                                                                                                                                          |
-| Email       | • Must be valid email format (example@domain.com)                                                                                                                  |
+| Field       | Requirements                                                                                                                                           |
+|-------------|--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Name        | • Letters, numbers, and spaces only<br>• Cannot be blank                                                                                               |
+| Phone       | • Minimally 3 digits                                                                                                                                   |
+| Email       | • Must be valid email format (example@domain.com)                                                                                                      |
```

Comment:
should we leave out the `.com` since `example@domain` is still a valid email in our case

Reply:
I think we can get away with it; it's quite grey area what is and is not a valid email, and if we omit the `.com` it might attract attention actually